### PR TITLE
Fixes #583 - Reworked object conditions

### LINKED ISSUE #583: Rework section 'basic object conditions'

### PR DESCRIPTION
**Pull request by ralf401**

### CODE REVIEW

File: misc/object-conditions/basics.rst [hunk 1]
```diff
@@ -1,443 +1,438 @@
...
-          ticket.*
-      * | not set (not defined)
-
-   .. container:: cfloat-clear
-
-      X
-
-Tags *(Scope: Ticket)*
-   Additional tags can be present in the ticket without issues.
+Object Conditions
```
Comment (mgruner) [inline misc/object-conditions/basics.rst:1]: For the parent page, "Getting started" should be title cased

File: misc/object-conditions/basics.rst [hunk 2]
```diff
@@ -1,443 +1,438 @@
...
+     than once
 
-.. hint::
-   **Differences in input fields**
+*has changed* conditions:
+   * Are only available for ticket attributes
+   * Don't affect ticket articles attributes
+   * Don't affect organization attributes
+   * Don't affect user attributes
+   * Don't affect group attributes
```
Comment (mgruner) [inline misc/object-conditions/basics.rst:432]: You could consider merging the don't affect entries into one line (with a comma separated list)

**Review by mgruner — APPROVED** [review]
Very nice change. Page is much more concise.

File: misc/object-conditions/basics.rst (hunk 1 above)
Comment (mgruner) [reply misc/object-conditions/basics.rst:1]: Also the "Object conditions" itself should be title cased.

Comment (ralf401) [reply misc/object-conditions/basics.rst:1]: Thanks for the hint! I reworked the page as well and removed the `Getting started` section.

**Pull request merged**